lib/operations/common/GitlabRepoRef.ts

```diff
@@ -34,43 +42,24 @@ export class GitlabRepoRef extends AbstractRemoteRepoRef {
         if (params.sha && !params.sha.match(GitShaRegExp.pattern)) {
             throw new Error("You provided an invalid SHA: " + params.sha);
         }
-        const result = new GitlabRepoRef(params.owner, params.repo, params.sha, params.rawApiBase, params.gitlabRemoteUrl, params.path,
-            params.branch);
-        return result;
+        return new GitlabRepoRef(params.owner, params.repo, params.sha, params.rawApiBase, params.gitlabRemoteUrl, params.path, params.branch);
     }
 
-    private static concatUrl(base: string, segment: string): string {
-        if (base.endsWith("/")) {
```

Comment:
Are these checks for "/" not needed?

Reply:
Correct.  I opted to fully construct the URLs using template strings and remove the need for this private method.  This is keeping in the style of other RepRef implementations.